src/PoshGitTypes.ps1

```diff
@@ -44,11 +44,41 @@ class PoshGitCellColor {
         return $str
     }
 
+    [string] ToEscapedString() {
```

Comment:
Does it make sense for `PoshGitCellColor` to have a `RenderAnsi()`, such that this function is essentially `EscapeAnsiString $this.RenderAnsi()`? That function could then be used in `PoshGitTextSpan`.

Reply:
Also, would `ToAnsiString()` be a better name than `RenderAnsi()`?

Reply:
That's what I get for coding and watching the playoffs.  :-)  Renamed `RenderAnsi()` to `ToAnsiString()`.   Can we do the `[PoshGitCellColor]` tweaks - ToAnsiString() and ToEscapedString() simplification in another PR?

Reply:
Yeah, that works. While you're at it, fix the typo in `EscapseAnsiString`. 😏

Reply:
Doh! Yeah, will do.